src/pages/home/ReportScreen.js

```diff
@@ -214,6 +215,10 @@ function ReportScreen({
         );
     }
 
+    if (reportID === '') {
```

Comment:
I don't think this condition is enough. While report data is loading, skeleton should also be displayed.

Please check this video:

https://github.com/Expensify/App/assets/108292595/e7471f26-dbeb-41ac-b81a-5900d1c850fe

Reply:
@rushatgabhane as you see, there's a brief moment when header is empty but report shows skeleton
(around 00:06s)

<img width="590" alt="Screenshot 2023-10-26 at 10 32 37 PM" src="https://github.com/Expensify/App/assets/108292595/1cde9546-9ca6-4636-9f38-0b9419c00fb8">